.github/workflows/project-tests.yml

```diff
@@ -4,31 +4,35 @@ on:
   push:
     branches: [ main ]
     paths:
-      - '**'
-      - '!.gitignore'
-      - '!**.DotSettings'
-      - '!**.md'
-      - '!.github/**'
-      - '!terraform/**'
+      - '**.cs'
+      - '**.cshtml'
+      - '**.sql'
+      - '**.csproj'
+      - '**.sln'
+      - '**/appsettings*json'
+      - '.config/dotnet-tools.json'
+      - 'stryker-config.json'
```

Comment:
Thinking on it, I don't think we need this workflow to trigger on a push to main - it only did it before to update the SonarQube report